internal/importer/service.go

```diff
@@ -623,7 +816,17 @@ func (s *Service) processNzbItem(ctx context.Context, item *database.ImportQueue
 		basePath = filepath.Join(basePath, *item.Category)
 	}
 
-	return s.processor.ProcessNzbFile(ctx, item.NzbPath, basePath, int(item.ID))
+	// Determine if max connections override is needed
+	var maxConnectionsOverride *int
```

Comment:
Why do you want to override the connections here? This can be dangerous and a potentialy cpu and memory leak